.github/workflows/build.yml

```diff
@@ -93,7 +93,7 @@ jobs:
         with:
           submodules: recursive
```

Comment:
I think you forgot `- uses: ./.github/actions/set-make-job-count` here

Reply:
Fixed and squashed.